src/test/ProtocolFeesImplementation.sol

```diff
@@ -33,4 +33,29 @@ contract ProtocolFeesImplementation is ProtocolFees {
     function updateProtocolFees(Currency currency, uint256 amount) public {
         ProtocolFees._updateProtocolFees(currency, amount);
     }
+
+    function consumeGasLimitAndFetchFee(PoolKey memory key) public {
+        // consume gas before calling fetchProtocolFee / getting the protocolFeeForPool from the controller
+        while (true) {
```

Comment:
Seems like these functions are the same -- could just write one and have that value as an input i think

Reply:
Hello, I'm not part of your team, but I have a question. Doesn't this method increase the testing time?